<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Анализ ФРТ</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QSplitter" name="m_Splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Изображение ФРТ</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QToolButton" name="m_PB_Zoom_Out">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string> - </string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3">
           <widget class="QToolButton" name="m_PB_Center">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>В центр</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QToolButton" name="m_PB_Zoom_In">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="m_PB_Zoom_Default">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Сбросить масштаб</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="QGraphicsView" name="m_GV_Image"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_GB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Параметры</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0" rowspan="2" colspan="4">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="m_LB_Magnification">
               <property name="text">
                <string>Увеличение системы:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="m_LE_Value_of_division"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="m_LB_Value_of_division">
               <property name="text">
                <string>Цена деления (мкм):</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="m_LE_Magnification"/>
             </item>
             <item row="2" column="2" colspan="2">
              <widget class="QLabel" name="m_LB_ResPixel">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Истинная цена деления ??? (мкм)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Вычислить</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="3">
              <widget class="QLineEdit" name="m_LE_EnergySize"/>
             </item>
             <item row="1" column="1">
              <widget class="QRadioButton" name="m_RB_CalcRadius">
               <property name="text">
                <string>Размер области (пикс) по количеству энергии (%):</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="m_LE_AreaSize"/>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="m_RB_CalcEnergy">
               <property name="text">
                <string>Количество энергии (%) по размеру области (пикс)
(радиус круга или половина стороны квадрата):</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0" colspan="4">
           <widget class="QGroupBox" name="m_GB_Center">
            <property name="title">
             <string>Координаты центра области</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="2" column="3">
              <widget class="QLabel" name="m_LB_Center_Y">
               <property name="text">
                <string>Y=</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="m_RB_CenterSet">
               <property name="text">
                <string>Задавать:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="m_LE_Center_X"/>
             </item>
             <item row="1" column="0" colspan="5">
              <widget class="QRadioButton" name="m_RB_CenterEnergy">
               <property name="text">
                <string>Вычислять по максимуму энергии (отдельно для круга и квадрата)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <widget class="QLineEdit" name="m_LE_Center_Y"/>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="m_LB_Center_X">
               <property name="text">
                <string>X=</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="3">
              <widget class="QRadioButton" name="m_RB_CenterMass">
               <property name="text">
                <string>Вычислять по центру масс</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="5">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="m_PB_Open">
            <property name="text">
             <string>Открыть файлы и вычислить</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="m_PB_Calc_Energy">
            <property name="text">
             <string>Вычислить выделенное</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="m_CB_FullTable">
            <property name="text">
             <string>Подробная таблица</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="m_PB_Calc_Avarage">
            <property name="text">
             <string>Усреднить выделенное</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QGroupBox" name="m_GB_Area">
            <property name="title">
             <string>Область</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="1" column="0">
              <widget class="QRadioButton" name="m_RB_Square">
               <property name="text">
                <string>Квадрат</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="m_RB_Circle">
               <property name="text">
                <string>Круг</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="m_Clear">
            <property name="text">
             <string>Очистить таблицу</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QPushButton" name="m_Print">
            <property name="text">
             <string>Вывести отчет в файл</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Результат вычислений</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="2">
        <widget class="QTableWidget" name="m_TW_Res">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Номер</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>имя файла</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>кол-во
энергии
 (%)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>центр X </string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>центр Y</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>размер
области</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>область</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Процент пересвета (%)</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
